client/src/components/AddQuestionForm.tsx

```diff
@@ -0,0 +1,156 @@
+      return;
+    }
+
+    const answersToSend: AnswerSendProp = {};
+    filteredData.answers.forEach((answer) => {
+      answersToSend[answer.answer] = +answer.numberOfResponse;
+      console.log("answersToSend", answersToSend);
+    });
+
+    const sendObject = {
+      question: filteredData.question,
+      answers: answersToSend,
+    };
+
+    try {
+      const response = await adminQuickRound.createQuestion(sendObject);
+
+      console.log("Question created successfully:", response);
+      queryClient.invalidateQueries({ queryKey: ["questions"] });
+      setCorrectScore(true);
+      closeModal();
+    } catch (error) {
+      console.error("Failed to create question:", error);
+    }
+  };
+
+  if (isSubmitting) return <Spinner />;
+
+  return (
+    <div className="flex justify-center items-center h-auto w-full">
+      <div className="p-12 bg-opacity-60 w-full max-w-3xl mx-auto">
+        <div className="mb-6">
+          <label className="block text-gray-700 mb-2">Number of Answers</label>
+          <span className="text-red-500">
+            {correctScore ? "" : "Total votes must be 100"}
+          </span>
+          <input
+            type="number"
+            min="1"
+            max="10"
+            value={numOfAnswers}
+            onChange={handleAnswerChange}
+            className="block w-full max-w-md bg-transparent outline-none border-b-2 border-black py-2 px-4"
+            placeholder="Enter number of answers"
+          />
+        </div>
+
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className="w-full max-w-3xl mx-auto"
+        >
+          <FormInputField
+            type="text"
+            register={register("question", { required: true })}
+            label="Question"
+            error={errors.question}
+            placeholder="Enter your question"
+          />
+
+          {[...Array(numOfAnswers)].map((_, index) => (
+            <div key={index} className="flex gap-4 mb-6">
+              <FormInputField
+                type="text"
+                register={register(`answers.${index}.answer`, {
+                  required: true,
+                })}
+                label={`Answer ${index + 1}`}
+                error={errors.answers?.[index]?.answer}
+                placeholder="Enter answer"
+                className="flex-1"
+              />
+              <FormInputField
+                type="number"
+                register={register(`answers.${index}.numberOfResponse`, {
+                  required: true,
+                })}
+                label={`Number of Votes ${index + 1}`}
+                error={errors.answers?.[index]?.numberOfResponse}
+                placeholder="Enter number of votes"
+                className="flex-1"
+              />
+            </div>
+          ))}
+
+          <Button type="submit" addNewContent="w-full mt-4">
+            Submit
+          </Button>
+        </form>
+      </div>
+    </div>
+  );
+}
```

Comment:
there’s no feedback for users if they input invalid data. display validation error messages next to the corresponding fields. should be like this: 

{errors.answers?.[index]?.numberOfResponse && (
  <p className="text-red-500">Number of points is required.</p>
)}

Reply:
The error prevents the user/admin from submitting the number of points if it is not specified in the form.